### PR TITLE
Artwork and recipient changes

### PR DESCRIPTION
**Pull request by DuongTyler**
This is what it looks like now:
image

### CODE REVIEW

**Comment by DuongTyler** [comment]
basically, artwork cards now use the full space available for art, and the translate button no longer stabs into the translated text

File: frontend/src/components/gallery/artworkCard/artworkCard.css [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
+    width: 100%;
+    height: 100%;
     position: relative;
     transition: all 0.4s ease;
 }
 
 .artwork-card-img > * {
     width: 100%;
-    height: auto;
+    /*height: auto;*/
```
Comment (zapplecat) [inline frontend/src/components/gallery/artworkCard/artworkCard.css:21]: Maybe just delete this.

**Review by zapplecat — APPROVED** [review]

**Review by zapplecat — CHANGES_REQUESTED** [review]
Actually, small issue with art. It's cutting into the card header.

image

Another example with screen dimensions
image

File: frontend/src/components/gallery/artworkCard/artworkCard.css (hunk 1 above)
Comment (DuongTyler) [reply frontend/src/components/gallery/artworkCard/artworkCard.css:21]: oops

**Comment by DuongTyler** [comment]
> Actually, small issue with art. It's cutting into the card header.
> 
> image
> 
> Another example with screen dimensions
> image

This doesn't happen on my build of this branch. I also fixed some backend problems that I'll push out in just a sec

**Review by zapplecat — APPROVED** [review]
We can fix the header stuff later. Lets just get the messages and other things loaded, then we can worry about looks.

**Pull request merged**